testcases/crypto/aes_func.c

```diff
@@ -2034,6 +2326,11 @@ int main(int argc, char **argv)
 
     pkey = CK_FALSE;
     rv = aes_funcs();
+
+    pkey = CK_TRUE;
```

Comment:
Ah OK, here it is, OK.

Could you skip this pkey= TRUE case for non-secure key tokens?  I.e. run it only for ep11 token for now.

Reply:
And like Juergen mentioned, we need a way for a failing testcase to tell if it was running with PKEY or without. As best the testcase name should be different .

Reply:
Yes, this is a good point: done.